cleaner/scheduled_tasks/classes/form/task_form.php

```diff
@@ -0,0 +1,107 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * @package     cleaner_scheduled_tasks
+ * @copyright   2019 Catalyst IT Australia {@link http://www.catalyst-au.net}
+ * @license     http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace cleaner_scheduled_tasks\form;
+use html_writer;
+use moodleform;
+
+defined('MOODLE_INTERNAL') || die();
+
+global $CFG;
+require_once("{$CFG->libdir}/formslib.php");
+
+class task_form extends moodleform {
+    protected function definition() {
+        $mform = $this->_form;
+        $tasks = $this->_customdata;
+
+        if (!$tasks) {
+            throw new \coding_exception(get_string('noscheduledtasks', 'cleaner_scheduled_tasks'));
+        }
+        get_string('pluginname', 'cleaner_scheduled_tasks');
+        // Display a header on the page.
+        // Need to put lang strings into lang folder later.
```

Comment:
Looks like you've already set up a lang strings file so this comment could be removed.